src/Modules/UngradedModules/SubquoModuleElem.jl

```diff
@@ -943,9 +945,7 @@ Return the `i`th generator of `M`.
 """
 function gen(M::SubquoModule{T}, i::Int) where T
   R = base_ring(M)
-  v::SRow{T} = sparse_row(R)
-  v.pos = [i]
-  v.values = [R(1)]
+  v = sparse_row(R, [i], [R(1)]; sort=false)
```

Comment:
```suggestion
  v = sparse_row(R, [i], [R(1)])
```
no longer needed due to https://github.com/thofma/Hecke.jl/pull/1502